### ISSUE TITLE
Fix memory leak when using PebblesDB with small key-value pairs

### ISSUE THREAD

**Issue by vijay03** [utsaslab/pebblesdb#24]
PebblesDB has a known memory-leak when used with a large number of small key-value pairs. This doesn't appear for some reason when PebblesDB is used with large key-value pairs (the default). This doesn't affect default behavior, but we would like to fix this going forward.

**Comment by lurais** [comment]
Hello, has this been fixed now? @vijay03

**Comment by vijay03** [comment]
No, not yet. I want to note that the problem doesn't appear for the default key-value pair sizes, so if you use those you will be fine.

**Comment by lurais** [comment]
> default key-value pair sizes

hello, Is there any detail info about default key-value pair sizes ?

**Comment by vijay03** [comment]
8 byte keys and 1024 byte values. The details are in the paper linked into the README.
